config/locales/provider_authorisation.yml

```diff
@@ -0,0 +1,17 @@
+en:
+  provider_authorisation:
+    errors:
+      make_decisions:
+        requires_provider_user_permission: You do not have the required user level permissions to make decisions on applications.
```

Comment:
Should we highlight you don't have the required permission for a specific provider? This could done by adding a generic `for this provider` phrase.

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/4211/commits/204c7a2dcbf27493229ac93ce8a07f8b38e8851a